tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  future: {
+    // removeDeprecatedGapUtilities: true,
+    // purgeLayersByDefault: true,
+  },
+  purge: [
+    './public/**/*.html',
+    './public/**/*.js',
```

Comment:
The purge whitelist directories are set here as of `tailwind 1.8`, instead of in the `postcss.config.js` this removes the need to add any conditional concerning the `NODE_ENV`